### PR TITLE
Update FixBookUI.java

### PR DESCRIPTION
**Pull request by chandrika-katikam**

### CODE REVIEW

File: src/library/fixbook/FixBookUI.java [hunk 1]
```diff
@@ -7,15 +7,15 @@
...
 	
-	public FixBookUI(fIX_bOOK_cONTROL CoNtRoL) {
-		this.CoNtRoL = CoNtRoL;
-		InPuT = new Scanner(System.in);
+	public FixBookUI(fIX_bOOK_cONTROL Control) {
+		this.CoNtRoL = Control;
+		input = new Scanner(System.in);
 		StAtE = uI_sTaTe.INITIALISED;
-		CoNtRoL.SeT_Ui(this);
+		Control.SeT_uI(this);
```
Comment (ttawan) [inline src/library/fixbook/FixBookUI.java:18]: This one too ```Control.setUI```

File: src/library/fixbook/FixBookUI.java [hunk 2]
```diff
@@ -72,7 +72,7 @@ public void RuN() {
 	
 	private String iNpUt(String prompt) {
```
Comment (ttawan) [inline src/library/fixbook/FixBookUI.java:73]: Change this to ```private String input(String prompt)```

File: src/library/fixbook/FixBookUI.java [hunk 3]
```diff
@@ -25,14 +25,14 @@ public void SeT_StAtE(uI_sTaTe state) {
 
 	
 	public void RuN() {
```
Comment (ttawan) [inline src/library/fixbook/FixBookUI.java:27]: Change this to ```run```

File: src/library/fixbook/FixBookUI.java [hunk 4]
```diff
@@ -7,15 +7,15 @@
 	public static enum uI_sTaTe { INITIALISED, READY, FIXING, COMPLETED };
 
 	private fIX_bOOK_cONTROL CoNtRoL;
```
Comment (ttawan) [inline src/library/fixbook/FixBookUI.java:9]: Change this to ```fixBookControl Control```

File: src/library/fixbook/FixBookUI.java [hunk 5]
```diff
@@ -7,15 +7,15 @@
 	public static enum uI_sTaTe { INITIALISED, READY, FIXING, COMPLETED };
```
Comment (ttawan) [inline src/library/fixbook/FixBookUI.java:7]: Change this to ```UI_State```

File: src/library/fixbook/FixBookUI.java [hunk 6]
```diff
@@ -7,15 +7,15 @@
 	public static enum uI_sTaTe { INITIALISED, READY, FIXING, COMPLETED };
 
 	private fIX_bOOK_cONTROL CoNtRoL;
-	private Scanner InPuT;
+	private Scanner input;
 	private uI_sTaTe StAtE;
```
Comment (ttawan) [inline src/library/fixbook/FixBookUI.java:11]: ```private UI_State State```

**Comment by aatish-rajkarnikar** [comment]
@chandrika-katikam  Please update the files

**Review by parneetbatra — APPROVED** [review]
All good

**Pull request merged**